be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -102,14 +102,13 @@ class ColumnWriter {
     struct Page {
         int32_t first_rowid;
         int32_t num_rows;
-        Slice null_bitmap = Slice((uint8_t*)nullptr, 0);
-        Slice data = Slice((uint8_t*)nullptr, 0);
+        Slice null_bitmap = Slice((const uint8_t*)nullptr, 0);
+        OwnedSlice data = OwnedSlice((uint8_t*)nullptr, 0);
```

Comment:
why not keep null_bitmap and data use same type?

Reply:
null_bitmap produced by NullBitmapBuilder,and NullBitmapBuilder didn't implement PageBuilder,here just keep NullBitmapBuilder's origin Semantics.